lib/Rex/Commands/File.pm

```diff
@@ -21,7 +21,7 @@ With this module you can manipulate files.
    }
    $fh->close;
  };
- 
+
```

Comment:
Looks like you changed all the POD sections fro some reason. I beleive it was just changed by your editor automatically. Can you send a commit that does not remove those spaces?